### PR TITLE
Ide 3061

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java [hunk 1]
```diff
@@ -81,7 +82,6 @@
         "com.liferay.portal.service.http",
         "com.liferay.portal.service.permission",
         "com.liferay.portal.servlet",
-        "com.liferay.portal.servlet.filters.authverifier",
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java]: Why did you remove all of these?

Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java]: For example, if we have:
"com.liferay.portal.servlet",
"com.liferay.portal.servlet.filters.authverifier",
that means all sub-packages of "com.liferay.portal.servlet" will be ignore, included "com.liferay.portal.servlet.filters.authverifier". so we no need to add the sub-packages if we already have the parent/root package in our list

Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java]: Btw this change has no relationship with code upgrade tool, I just reviewed the code and found it.

Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java]: Sounds good, thanks for explaination.

**Comment by gamerson** [comment]
Merged, thanks! See e28b151...3388b01 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
